lib/data/rates/statutory_sick_pay.yml

```diff
@@ -25,6 +25,6 @@
   ssp_weekly_rate: 87.55
 
 - start_date: 2014-04-06
-  end_date: 2015-04-05
+  end_date: 2017-04-05
```

Comment:
Nice one @ikennaokpala. This was specified by the content team. Me feeling is because it is not very likely to change in two years. Should be good to go.